openaerostruct/docs/advanced_features/custom_mesh_example.rst

```diff
@@ -21,7 +21,8 @@ The following shows a visualization of the mesh.
 The complete script for the optimization is as follows.
 Make sure you go through the :ref:`Aerostructural_with_Wingbox_Walkthrough` before trying to understand this setup.
 
-.. literalinclude:: /advanced_features/scripts/two_part_wing_custom_mesh.py
+.. embed-code::
+   scripts/two_part_wing_custom_mesh.py
```

Comment:
I don't see this one in the rendered docs online

Reply:
I'm not sure what's happening here.  Might go back to the `literalinclude` if it doesn't show up.

Reply:
Is it because of how you've defined the path? It looks like all other instances define the path using the python import syntax with dots

Reply:
Maybe, but that's how it was defined in the docs that used the extensions previously.  See this [PR](https://github.com/mdolab/OpenAeroStruct/commit/e2e3c848687d73710267ef1e30b125f404aa20f3)

Reply:
Oh wait, they had the scripts in the same directory before.  Maybe I do need to use the python import syntax